partiql-test-data/fail/static-analysis/primitives/operator/is-operator.ion

```diff
@@ -1,23 +1,31 @@
-parse::{
+{
+    name: "IS operator expects type name",
+    statement: "NULL IS `null`",
+    assert: {
+        result: StaticAnalysisFail
+    },
+}
+
+{
     name: "IS NOT operator expects type name",
     statement: "NULL IS NOT `null`",
     assert: {
-        result: ParseError
+        result: StaticAnalysisFail
```

Comment:
These tests were previously considered `parse`/`syntax` failure tests. Based off this [comment](https://github.com/partiql/partiql-tests/issues/16#issuecomment-1165028166) and other discussions, these tests correspond more so to `static-analysis` failures.